genesis/utils/array_class.py

```diff
@@ -2004,6 +2005,70 @@ def __init__(self):
         return ClassEntitiesState()
 
 
+# =========================================== StaticRigidSimConfig ===========================================
+
+
+@dataclasses.dataclass
+class StaticRigidSimDummy:
+    para_level: int = dataclasses.field(metadata={FIELD_METADATA_CACHE_VALUE: True})
```

Comment:
I can see why you're going to ask me to make it possible to set `FIELD_METADATA_CACHE_VALUE` conceptually on an entire dataclass at some point :)

Reply:
I don't fully get it. Why is it better than the current implementation?

Reply:
done